docs/content/howto/optimize-chunks.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Optimize chunk count
+order: 10
+---
+
+
+## Understanding chunks and their impact on performance
+
+Rerun stores all its data in Chunks -- Arrow-encoded tables of data. A basic understanding of chunks is key to understanding how Rerun works and why performance behaves the way it does. See the [chunk concept documentation](../concepts/chunks.md) for more details.
+
+Chunks are the atomic unit of work in Rerun: the performance cost of logging, ingesting, storing, querying, and visualizing data (including memory overhead to some extent) scales roughly linearly with the number of chunks in the system (ignoring caching and indexing optimizations). Many small chunks cause significantly more overhead than fewer, larger ones. Larger chunks reduce index pressure and per-chunk overhead, improving write and query throughput.
```

Comment:
I do wonder if the visualizing part takes it a bit too far. most visualizing happens behind a cache and we deal with "virtual" chunks effectively